pkg/gen/nodejs-templates/helm/v2/helm.ts

```diff
@@ -189,10 +193,19 @@ export class Chart extends yaml.CollectionComponentResource {
                 // > is done.
                 const release = shell.quote([releaseName]);
                 const values = path.quotePath(overrides.name);
+                const apiVersionsArgs = cfg.apiVersions
+                  ? cfg.apiVersions.length > 1
+                    ? `--api-versions={${cfg.apiVersions
+                        .map(apiVersion => shell.quote([apiVersion]))
+                        .join(',')}}`
+                    : `--api-versions=${cfg.apiVersions
```

Comment:
It looks like the length would only be 1 here, so how about
```ts
`---api-versions=${shell.quote(cfg.apiVersions[0])}`
```

Reply:
Thanks for the suggestion. `shell.quote` takes a `string[]` so I just passed in `apiVersions`.